plugins/node/opentelemetry-instrumentation-redis/examples/package.json

```diff
@@ -37,6 +37,7 @@
     "@opentelemetry/instrumentation": "^0.32.0",
     "@opentelemetry/instrumentation-http": "^0.32.0",
     "@opentelemetry/instrumentation-redis": "^0.32.0",
+    "@opentelemetry/redis-common": "^0.33.0",
```

Comment:
Since this example package already depends on `@opentelemetry/instrumentation-redis`, no need to add this dependency here